packages/gatsby/src/utils/worker/render-html.ts

```diff
@@ -20,16 +23,27 @@ export const renderHTML = ({
     paths,
     path =>
       new Promise((resolve, reject) => {
+        // Make sure we get the latest version during development
+        if (stage === `develop-html`) {
+          delete require.cache[require.resolve(htmlComponentRendererPath)]
+        }
         const htmlComponentRenderer = require(htmlComponentRendererPath)
         try {
-          htmlComponentRenderer.default(path, (_throwAway, htmlString) => {
-            resolve(
-              fs.outputFile(
-                getPageHtmlFilePath(join(process.cwd(), `public`), path),
-                htmlString
-              )
-            )
-          })
+          htmlComponentRenderer.default(
+            path,
+            async (_throwAway, htmlString) => {
```

Comment:
What's the point of this arrow being `async`? (Unless the callback to `htmlComponentRenderer` simply has to be a promise, but it didn't before this change). Feels like this arrow wants to either `await` something or return an explicit promise, or shouldn't be `async`.

Reply:
yeah not sure why it changed. Reverting.